README.md

```diff
@@ -23,20 +23,57 @@ subsembly
 
 Main types and API entries are defined in `runtime` folder. `runtime.ts` file in `runtime` folder defines types and constants for the frame modules and pallets.
 
-### Adding pallets
+### Developing Subsembly Runtimes
 
-This starter project comes with `Aura`, `Balances` and `Timestamp` pallets. The minimal requirements for building and running account-based runtime on Substrate node is to have those three pallets.
+#### Runtime configuration
 
-In order to add pallets to your runtime, place your implementation of pallet inside `/pallets` folder and implement runtime API entries for the pallet.
+Top-level runtime folder consists of Runtime API entries that should be exposed according to Polkadot Host Specification. There is also, `runtime.ts` file, where types and constants for the frame modules and pallets are defined. We define general types that are used across the runtime and also pallet specific constants and types. 
```

Comment:
it's not actually Polkadot Host Spec. Atm we are implementing the interface of the runtime API for Substrate Hosts. Maybe we should use Substrate Hosts and not Polkadot?